app/posts/register-trainee-teachers/2025-04-01-communicating-api-and-csv-changes.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Communicating API and CSV changes
+description: Informing the sector of changes to the Register API and CSV
+date: 2025-04-01
+tags:
+  - trainees
+  - providers
+  - vendors
+  - API
+  - CSV
+screenshots:
+  items:
+    - text: Proposed footer with news and updates link
+      src: proposed-footer.png
+    - text: Proposed news and updates list
+      src: news-and-updates-index.png
+    - text: Proposed news and updates post item
+      src: news-and-updates-post.png
+    - text: How to use this service
+      src: how-to-use-this-service.png
+    - text: Navigation exploration
+      src: navigation-exploration.png
+    - text: Existing footer and link to the news page
+      src: existing-footer-and-link-to-news.png
+    - text: Existing news and updates page
+      src: existing-news-and-updates-page.png
+---
+
+Register currently sends email notifications to providers, and we have a basic news and updates page for the service; however, this needs to be enhanced to accommodate our new role in communicating changes to the sector.
+
+## What we did
+
+We began by examining the collection of guidance, news, and updates currently available on Register. While functional, we wondered whether we could integrate it more cohesively within the service.
+
+It became clear that the existing single page for news and updates would not be sufficient in the future.
+
+In terms of the navigation, we investigated adding a 'Documentation' link in the main header and then using secondary navigation to access 'News and updates', 'How to use this service', 'API documentation' and 'CSV documentation'.
+
+![Screenshot showing an alternative navigation](navigation-exploration.png "Exploration of navigation options")
+
+This proved unviable though because;
+
+* The documentation isn't available to users who aren't logged into the service
+* This header version has been deprecated for a separate [service navigation component](https://design-system.service.gov.uk/components/service-navigation/) which we're adopting.
```

Comment:
```suggestion
This proved unviable because:

- the documentation is not available to users who are logged out of the service
- the header navigation has been deprecated for a separate [service navigation component](https://design-system.service.gov.uk/components/service-navigation/), which we’re adopting.
```